index.html

```diff
@@ -162,25 +162,39 @@ <h2 class="salutation">Tonic</h2>
             <li><a href="https://twitter.com/home"><img src="images/Vector (2).jpg" alt="twitter"></a></li>
         </ul> 
     </section>  
+    <div>
+        <button class="get-resume" type="button">Get my Resume</button>
 
-            
-</div>
-<button class="get-resume" type="button">Get my Resume</button>
-</div>
-<section class="skills">
-
-<ul class="about-skill">
-    <li class="aboutl">Languages <img src="images/arrowDown.jpg" alt="">
-    </li> 
-    <p><img src="images/js.jpg" alt="">JavaScript</p>
-    <p><img src="images/html.jpg" alt="">HTML</p>
-    <p><img src="images/ccs3.jpg" alt="">CSS</p>
+    </div>
     
-    <li class="about">Frameworks <img src="images/arrowRight.jpg" alt=""></li> 
-    <hr>
-    <li class="about">Skills <img src="images/arrowRight.jpg" alt=""></li> 
-    <hr>
-</ul>
+    <section class="skills">
 
-    
-</section>
\ No newline at end of file
+        <ul class="about-skill">
+            <li class="aboutl">Languages <img src="images/arrowDown.jpg" alt=""></li> 
+            <p><img src="images/js.jpg" alt="">JavaScript</p>
+            <p><img src="images/html.jpg" alt="">HTML</p>
+            <p><img src="images/ccs3.jpg" alt="">CSS</p>
+            
+            <li class="about">Frameworks <img src="images/arrowRight.jpg" alt=""></li> 
+            <hr>
+            <li class="about">Skills <img src="images/arrowRight.jpg" alt=""></li> 
+            <hr>
+        </ul>
+    </section>
+</section>
+
+<!-- This section is about contact form -->
+<section class="contact-form">
+    <h2 class="contact-me ">Contact me</h2>
+    <p class="contact-item">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+        I’d love to help with it!</p>
+        <form action="#" class="form contact-item">
```

Comment:
- [ ] Good job so far 👍 However, please make sure to collect data by using the `formspree service` according to the project requirements (mentioned below)👇 and form submits successfully. Right now, the form is not submitting ⚠ For that, kindly ensure that the form uses the `action` and `method` attribute to integrate with [formspree service](https://formspree.io/html) such that
  - `action` value should be similar to `https://formspree.io/f/{form_id}`
  - `method` value equals `POST`.
- You can go through [this video tutorial](https://www.youtube.com/watch?v=sVPlb5C3oQ8) to learn about form submission using formspree.

  > _Collect data by using the [Formspree service](https://formspree.io/html)._